src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,94 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private int size;
+    private static final int defaultSize = 10;
```

Comment:
what about naming convention for the constants? and ordering? I wrote about it in previous comment.